spec/models/ad_spec.rb

```diff
@@ -1,32 +1,40 @@
 require 'rails_helper'
 
 describe Ad do
+  describe '.show_ads?' do
+    it 'shows the ad' do
+      expect(Ad.show_ads?).to eq(true)
+    end
+  end
 
-  context "with one ad created" do
+  describe '.ve_banner' do
+    let(:banner_ad_attrs) { { style: 'banner', client: 've', category: 'b2c_ab' } }
+    let(:banner_ad) { double('BannerAd', banner_ad_attrs) }
 
-    let(:ad){ FactoryGirl.create(:ad) }
+    before do
+      banner_ads = double
```

Comment:
can we take this out of the before block and put it into a let please? and format that like `let(:banner_ads) { double('BannerAds') }` to keep the style of our tests consistent.